### PR TITLE
Added Sprite::sized(custom_size)

### PR DESCRIPTION
**Pull request by Kees-van-Beilen**
# Objective
add a quick shorthand for creating a sprite with an custom size. This is often desired when working with custom units or scaled sprites and allows for a cleaner syntax in those cases/

## Solution

Implemented a `sized` function on the Sprite struct which creates a Sprite, sets the custom size and leaves the rest at their default values

---

## Changelog

- Added `Sprite::sized(custom_size: Vec2)`

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

**Review by NthTensor — APPROVED** [review]
If the author doesn't respond, it should be trivial to adopt this.

**Comment by alice-i-cecile** [comment]
I'll merge this once merge conflicts are fixed.

**Comment by alice-i-cecile** [comment]
Closing as abandoned, but do ping me / reopen if you want to redo this or resolve merge conflicts.

**Comment by NthTensor** [comment]
Yes, please do! Sorry for taking so long to review the first one.

**Pull request closed without merging**
